supabase/migrations/20241109133338_initial_structure.sql

```diff
@@ -10,14 +13,14 @@ create table "public"."image" (
     "id" bigint generated by default as identity not null,
     "category_id" bigint not null,
```

Comment:
rename "category_id" to "category", because we use "category" in production

Reply:
So wie abgesprochen werde ich das hier nicht fixen, sondern wir nutzen die Tabelle